components/external_storage/export/src/export.rs

```diff
@@ -333,17 +333,36 @@ impl ExternalStorage for EncryptedExternalStorage {
         &self,
         storage_name: &str,
         restore_name: std::path::PathBuf,
-        compressed_range: Option<(u64, u64)>,
+        range: Option<(u64, u64)>,
```

Comment:
There are too many arguments to the function, can we pack them into something like `RestoreConfig`?

Reply:
done